### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by tsisodia10**

### CODE REVIEW

File: feast_mariadb/Readme.md [hunk 1]
```diff
@@ -29,47 +29,135 @@ d. You should see both databases deployments
...
+    database: sampledb
     user: feast
     password: feast
 offline_store:
     type: postgres
     host: localhost
     port: 5432
-    database: feast
-    db_schema: feast
+    database: sampledb
```
Comment (dmartinol) [inline feast_mariadb/Readme.md:64]: Does it make sense to have the same database `sampledb` for both o line and offline store? (Just asking, with the goal of emulating a real world setup)

File: feast_mariadb/Readme.md [hunk 2]
```diff
@@ -29,47 +29,135 @@ d. You should see both databases deployments
...
+  conn.execute(insert_stmt)
+Deploying infrastructure for driver_hourly_stats
+Deploying infrastructure for driver_hourly_stats_fresh
 ```
 
-### Step 5: Do `feast apply` -- 
+`feast feature-views list`
+```
+(feast) ➜  feature_repo git:(mariadb) ✗ feast feature-views list;
+zsh: /usr/local/bin/feast: bad interpreter: /usr/local/opt/python@3.11/bin/python3.11: no such file or directory
```
Comment (dmartinol) [inline feast_mariadb/Readme.md:98]: Can we remove this `bad interpreter` message?

File: feast_mariadb/Readme.md (hunk 1 above)
Comment (jeremyary) [reply feast_mariadb/Readme.md:64]: My suspicion is that you'd be right in prod scenarios. That said, we'll be running some RH demos with shared DB's IIUC (thus the ask for Maria support) so good to know it works like this as well.

**Review by dmartinol — APPROVED** [review]
Lgtm apart a few minor comments

**Review by jeremyary — APPROVED** [review]

**Pull request merged**